app/assets/stylesheets/application.scss

```diff
@@ -7,3 +7,57 @@
 @import "base/base";
 
 @import "layout";
+
+// variables
+$alert: #de1b31;
+$error: #de1b31;
+$notice: #000;
+$success: #2adc55;
+
+.nav {
+
+    .menu {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Line should be indented 2 spaces, but was indented 4 spaces